package.json

```diff
@@ -615,6 +621,7 @@
     "compile": "tsc -p ./ && node ./scripts/createVersionNumber.js",
     "watch": "tsc -watch -p ./",
     "test": "npm run compile && mocha",
+    "test": "npm run compile && mocha",
```

Comment:
I guess this was a mistake but could you remove the duplicated line?

Reply:
Oh, sure. I'll remote it asap.